Xamarin.Forms.Platform.UAP/CarouselView/FormsViewToNativeUWP.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using Windows.Foundation;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace Xamarin.Forms.Platform.UWP
+{
+    public static class FormsViewToNativeUWP
+    {
+        public static FrameworkElement ConvertFormsToNative(Xamarin.Forms.View view, Rectangle size)
+        {
+            //var vRenderer = RendererFactory.GetRenderer (view);
```

Comment:
The biggest limitation i have in UWP is that DataTemplate doesn't support parameters like Xamarin.Forms DataTemplate does.

If UWP DataTemplate can support ```Func<object>``` as parameter, i wont need to render all the pages ahead of time. I will use a DataTemplateSelected instead and render the views on demand and use ItemsSource as FlipView source directly.